big_tests/tests/graphql_metric_SUITE.erl

```diff
@@ -359,7 +359,7 @@ check_node_result_is_valid(ResList, MetricsAreGlobal) ->
     [#{<<"key">> := <<"value">>,<<"value">> := V} | _] =
         maps:get([<<"global">>,<<"sm_unique_sessions">>,<<"count">>], Map),
     ?assert(is_integer(V)),
-    HistObjects = maps:get([<<"global">>, <<"xmpp_element_size_in">>, <<"byte_size">>], Map),
+    HistObjects = maps:get([<<"global">>, <<"xmpp_element_size_in">>, <<"c2s">>, <<"byte_size">>], Map),
```

Comment:
Why did these tests pass before if the name was incorrect? Information like this would be good to have in the PR.